LayoutTests/fast/storage/serialized-script-value.html

```diff
@@ -6,7 +6,7 @@
     <body>
         <script>
 
-const currentVersion = 0x0d;
+const currentVersion = 0x0e;
```

Comment:
Why is this in hex? If this needs to be kept in sync with the one in the `.cpp` file I would think we’d write 14 in both places.